core/src/main/scala/za/co/absa/spline/harvester/builder/OperationNodeBuilder.scala

```diff
@@ -46,3 +47,7 @@ trait OperationNodeBuilder {
 
   def outputExprToAttMap: Map[sparkExprssions.ExprId, Attribute]
 }
+
+object OperationNodeBuilder {
+  type Attributes = Seq[Attribute]
```

Comment:
I feel like this will only decrease the readability. I don't see any problems with using `Seq[Attribute]` as a type.

Reply:
It was primarily introduced to get rid of `Seq[Seq[Attribute]] `in other places.